ietf/templates/doc/ad_list.html

```diff
@@ -87,6 +103,37 @@ <h2 class="mt-5" id="{{ dt.type.0 }}">{{ dt.type.1 }} State Counts</h2>
     </script>
     <script src="{% static "ietf/js/highcharts.js" %}"></script>
     {{ data|json_script:"data" }}
+
+    <script>
+         Array.from(document.querySelectorAll("table"))
+            .filter(table => {
+                const hasDataSum = Boolean(table.querySelector("[data-sum]"))
+                return hasDataSum
+            })
+            .forEach(table => {
+                const sums = Array.from(table.querySelectorAll("[data-sum]")).reduce((
+                    sums,
```

Comment:
There are 2 variables named "sums" in 2 different scopes, which can cause confusion when reading.

Reply:
Fixed